db/migrate/20250722083933_add_gender_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddGenderToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :gender, :integer, default: 0
```

Comment:
```suggestion
    add_column :users, :gender, :integer
```
Bỏ default, nếu k sẽ bị sai data nếu không truyền dữ liệu